source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -35,18 +35,86 @@ message TransformationSequence {
 
 message Transformation {
   oneof transformation {
-    TransformationMoveBlockDown move_block_down = 1;
-    TransformationSplitBlock split_block = 2;
+    TransformationAddConstantBoolean add_constant_boolean = 1;
```

Comment:
I don't know much about protobufs, but are creating a versioning problem by renumbering these?

Reply:
Yes, I believe so - renumbering means that data that was previously serialised won’t be deserialisable. My thinking was that this could be ignored until the tool is fleshed out with a basic set of transformations (because before then there will not exist any valuable serialised data to be preserved), after which we could do a renumbering leaving some gaps for future transformations.

Reply:
Are the numbers important or are you just keeping a lexicographical ordering? Maybe don't change them and add a comment stating that new transforms should take the next available number at the end?

Reply:
I was just keeping a lexicographical ordering.  But really this isn't super important.  So in my latest commit I have changed things so that numerical ordering is used, with new transformations taking on successive numeric ids.  That will make it trivial to see what the next id should be when we have tens of transformation kinds, which is good.